docs/pct/README.md

```diff
@@ -0,0 +1,291 @@
+# Puppet Content Templates
+
+* [Overview](#overview)
+* [Getting Started](#getting-started)
+  * [pct new](#pct-new)
+  * [Template Updates](#template-updates)
+  * [More Templates](#more-templates)
+* [Writing Your Own Templates](#writing-templates)
+  * [Dos and Don'ts](#dos-and-donts)
+  * [Structure](#structure)
+    * [pct-config.yml](#pct-configyml)
+  * [Templating Language](#templating-language)
+  * [Testing](#testing)
+  * [Sharing](#sharing)
+* [Overriding Template Defaults](#overriding-template-defaults)
+
+## Overview
+
+`Puppet Content Templates` (PCT) allow you to quickly create Puppet content from scaffolding templates built by Puppet and the Community.
+
+> :warning: PCT is currently in an EXPERIMENTAL phase and feedback is encouraged via [pdk/discussions](https://github.com/puppetlabs/pdk/discussions) and starting a `feedback` post.
+
+## Getting Started
+
+Install the `exprimental` PCT release available from [github.com/puppetlabs/pdkgo/releases/](https://github.com/puppetlabs/pdkgo/releases/)
+
+Templates currently come in 2 flavours: `project` and `item`.
+
+* A `project` is a template containing many files in a particular structure. _These are great starting points._
+* An `item` is a template that will supplement a project or existing content.
+
+### pct new
```

Comment:
```suggestion
### Creating content using Puppet Content Templates
```

I would know to look for creating if I didn't know anything about the tool, where as I wouldn't know to look for `pct new`